demo/package.json

```diff
@@ -20,8 +20,9 @@
     "gatsby-remark-images": "^5.7.0",
     "gatsby-source-filesystem": "^3.10.0",
     "gatsby-transformer-sharp": "3.10.0",
+    "gatsby-transformer-json": "",
```

Comment:
we might want to put something like "^4.24.0" here instead of leaving it blank.  this will make it better for version control for us, and `4.24.0` is the version that currently gets installed when you run `yarn` so that's the latest version, if there's a major version bump we'll want to manage that

Reply:
also! once this is updated, you'll want to run `yarn` before pushing as we want to get the updated `yarn.lock` file as well